openapi.yaml

```diff
@@ -3062,6 +3062,857 @@ paths:
+      - name: instanceId
+        in: path
+        description: The ID of the Managed MySQL Database.
+        required: true
+        schema:
+          type: integer
+      - name: backupId
+        in: path
+        description: The ID of the Managed MySQL Database backup.
+        required: true
+        schema:
+          type: integer
+    get:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Backup View
+      operationId: getDatabasesMySQLInstanceBackup
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-backup-view
+      x-linode-grant: unrestricted only
+      description: |
+        Display information for a single backup for an accessible Managed MySQL Database on your Account.
+
+        Only unrestricted Users can access this command.
+
+        **Beta**: This endpoint is in **closed** beta. To get early access and test new Linode products before they
+        hit the market, join the [Linode Green Light](https://www.linode.com/green-light/) program.
+        When accessing this endpoint, please make sure to prepend all requests with `/v4beta` instead of `/v4`, and be
+        aware that this endpoint may receive breaking updates in the future. This notice will be removed when this out of
+        beta.
+      security:
+      - personalAccessToken: []
+      responses:
+        '200':
+          description: Returns a single backup for the Managed MySQL Database.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DatabaseBackup'
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+              https://api.linode.com/v4beta/databases/mysql/instances/123/backups/456
+      - lang: CLI
+        source: >
+          linode-cli databases mysql-backup-view 123 456
+  /databases/mysql/instances/{instanceId}/backups/{backupId}/restore:
+    x-linode-cli-command: databases
+    parameters:
+      - name: instanceId
+        in: path
+        description: The ID of the Managed MySQL Database.
+        required: true
+        schema:
+          type: integer
+      - name: backupId
+        in: path
+        description: The ID of the Managed MySQL Database backup.
+        required: true
+        schema:
+          type: integer
+    post:
+      tags:
+      - Databases
+      summary: Managed MySQL Database Backup Restore
+      operationId: postDatabasesMySQLInstanceBackupRestore
+      servers:
+      - url: https://api.linode.com/v4beta
+      x-linode-cli-action: mysql-backup-restore
+      x-linode-grant: unrestricted only
+      description: |
+        Restore a backup to a Managed MySQL Database on your Account.
+
+        The Database must have an `active` status to perform this command.
+
+        Only unrestricted Users can access this command.
+
```

Comment:
Here's a tricky one, Ben. Currently - SG is aware of this - there is a bug with restoring from backup if the customer has reset their credentials after the last backup. It will result in a failed cluster that we need to _try_ and fix.

Maybe you can put something in here about that? Hopefully it's temporary, and maybe this should be run up the stakeholder chain to see if we want to make it known publicly or not.